typescript/examples/flatbufferswriter/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@foxglove/mcap-example-flatbuffer-writer",
```

Comment:
let's have this match the dir name, so `typescript/examples/X` -> `@foxglove/mcap-example-X`. seems like other packages are doing this consistently already.